docs/guide/dev/features/autocomplete/README.md

```diff
@@ -14,4 +14,140 @@ For licensing, see LICENSE.md.
 
 <info-box info="">
     This feature was introduced in CKEditor 4.10. It is provided through optional plugins that are not included in the CKEditor presets available from the <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site and {@link guide/dev/plugins/README need to be added to your custom build} with <a href="https://ckeditor.com/cke4/builder">online builder</a>.
-</info-box>
\ No newline at end of file
+</info-box>
+
+This plugins provides smart completion feature for custom text matches based on user input. Every time when user type character he will get information about available, existing options.
+
+## Usage
+
+The autocomplete plugin shows dropdown with available options every time when user types matching text.
+
+{@img assets/img/autocomplete_01.png Using autocomplete to get ticket hints.}
+
+When you press <kbd>enter</kbd>, <kbd>tab</kbd> or any other customized [`commitKeystroke`](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.config#cfg-autocomplete_commitKeystrokes) suggested value will be inserted into editor. 
+
+## Configuration
+
+Autocomplete plugin utilizes two important callbacks which allows you to customize matching function and data source:
+
+* A text test callback.
+* A data callback.
+
+They are required to setup autocomplete instance which will be immediately attached into editor after its creation. Autocomplete can be configured by [`configDefinition`](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR_plugins_autocomplete.configDefition) object passed into autocomplete constructor.
```

Comment:
Typo `configDefition` -> `configDefinition`.

Reply:
Instead of hardcoding production links use our API docs link instead, like that one:

```
{@linkapi CKEDITOR.config.removePlugins configuration}
```